changelog.txt

```diff
@@ -31,7 +31,7 @@
 
 * Add `library_name` parameter to Sign in with Google button. See [#9924](https://github.com/google/site-kit-wp/issues/9924).
 * Add incremental ESLint rule to avoid defining `scenario.label` on stories. See [#9865](https://github.com/google/site-kit-wp/issues/9865).
-* Add option to enable "One-tap" sign-in across site when using Sign in with Google. See [#9861](https://github.com/google/site-kit-wp/issues/9861).
+* Add option to enable "One Tap" sign-in across site when using Sign in with Google. See [#9861](https://github.com/google/site-kit-wp/issues/9861).
```

Comment:
@aaemnnosttv, did you mean to amend the changelog?

Reply:
I did, yes. I agree we don't normally make updates to the changelog, but I don't think there is any reason not to in this case as it's not a meaning-altering change.